packages/react-ui/internal/icons/16px/index.tsx

```diff
@@ -21,10 +21,23 @@ const SvgIcon: React.SFC<SvgIconProps> = ({ color, size, style, children, ...res
 
 export const ArrowChevronDownIcon = (props: SvgIconProps) => (
   <SvgIcon {...props}>
-    <svg viewBox="0 0 16 16">
+    <svg viewBox="0 0 16 16" style={{ marginBottom: '-3px' }}>
```

Comment:
```suggestion
    <svg viewBox="0 0 16 16" style={{ marginBottom: '-0.1875em' }}>
```

Вероятно это то же смещение, что и в новых react-icons. Значение в em лучше, потому что масштабируется со шрифтом. В `ArrowChevronUpIcon` тоже.